rllib/core/learner/learner.py

```diff
@@ -306,147 +347,123 @@ def hps(self) -> LearnerHyperparameters:
         """The hyper-parameters for the learner."""
         return self._hps
 
-    def configure_optimizers(self) -> ParamOptimizerPairs:
-        """Configures the optimizers for the Learner.
-
-        Do not override this method for your custom algorithms (which require certain
-        optimizers), but rather override the `self.configure_optimizers_for_module(
-        module_id=..)` method and return those optimizers from there that you need for
-        the given module.
+    def register_optimizer(
```

Comment:
New API allowing for more flexibility in users overriding `configure_optimizers` OR `configure_optimizers_for_module`.

Reply:
Very nice API change. I love it.